### PR TITLE
Added cloud service NIC and PIP APIs specifications

### PR DESCRIPTION
**Pull request by richajain**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [X] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
 
### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [X] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

The review was already done on the private repo https://github.com/Azure/azure-rest-api-specs-pr/pull/1665 .

### CODE REVIEW

**Comment by lirenhe** [comment]
@richajain, could you take a look at the lint diff error?

**Comment by richajain** [comment]
> @richajain, could you take a look at the lint diff error?
Resolved

**Comment by lirenhe** [comment]
> > @richajain, could you take a look at the lint diff error?
> > Resolved

Could you also take a look at Avocado error, the api version in the json file is '"version": "2020-05-01"',  however, it is in the folder of 2020-07-01

**Comment by richajain** [comment]
> Could you also take a look at Avocado error, the api version in the json file is '"version": "2020-05-01"', however, it is in the folder of 2020-07-01
Done

File: specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/CloudServiceRoleInstancePublicIpList.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "cloudServiceName": "cs1",
+    "resourceGroupName": "cs-tester",
+    "api-version": "2018-10-01",
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/CloudServiceRoleInstancePublicIpList.json:5]: Is this value expected?

File: specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/CloudServiceNetworkInterfaceGet.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "api-version": "2018-10-01",
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/examples/CloudServiceNetworkInterfaceGet.json:3]: It looks this value is not correct.

**Review by lirenhe — APPROVED** [review]

**Pull request merged**